packages/@okta/vuepress-site/guides/sign-into-native-ui/handle-authentication-responses/iOS/handle-responses.md

```diff
@@ -0,0 +1,107 @@
+### Response handling
+
+To initiate a particular flow please make a call to one of the available functions in OktaAuthSdk.swift:
+
+```swift
+// Authentication
+public class func authenticate(with url: URL,
+                               username: String,
+                               password: String?,
+                               onStatusChange: @escaping (_ newStatus: OktaAuthStatus) -> Void,
+                               onError: @escaping (_ error: OktaError) -> Void)
+
+// Unlock account
+public class func unlockAccount(with url: URL,
+                                username: String,
+                                factorType: OktaRecoveryFactors,
+                                onStatusChange: @escaping (_ newStatus: OktaAuthStatus) -> Void,
+                                onError: @escaping (_ error: OktaError) -> Void)
+
+// Forgot password
+public class func recoverPassword(with url: URL,
+                                  username: String,
+                                  factorType: OktaRecoveryFactors,
+                                  onStatusChange: @escaping (_ newStatus: OktaAuthStatus) -> Void,
+                                  onError: @escaping (_ error: OktaError) -> Void)
+
+// Restore authentication
+public class func fetchStatus(with stateToken: String,
+                              using url: URL,
+                              onStatusChange: @escaping (_ newStatus: OktaAuthStatus) -> Void,
+                              onError: @escaping (_ error: OktaError) -> Void)
+```
+
```

Comment:
Since the previous step covered it, remove all of the above. This step should focus on writing the `handleStatus` and `handleError` functions that were called in the previous step.